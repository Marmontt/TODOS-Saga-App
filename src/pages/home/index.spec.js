import React from "react";
import { shallow } from "enzyme";

import HomePage from "./index";

describe("Home page", () => {
  it("should render properly", () => {
    const wrapper = shallow(<HomePage />);
    expect(wrapper).toMatchSnapshot();
  });
});
