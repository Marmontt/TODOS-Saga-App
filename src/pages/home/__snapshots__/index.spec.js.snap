// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should render properly 1`] = `<div />`;
